src/bp-forums/admin/functions.php

```diff
@@ -165,8 +165,10 @@ function bbp_sanitize_slug( $slug = '' ) {
 	$value = ltrim( $value, '/' );
 	$value = rtrim( $value, '/' );
 
+	$slug = remove_accents( apply_filters( 'bbp_sanitize_slug', $value, $slug ) );
```

Comment:
```suggestion
	return remove_accents( apply_filters( 'bbp_sanitize_slug', $value, $slug ) );
```

No need to take this in variable just return from here.